crates/blockifier/src/bouncer_test.rs

```diff
@@ -187,7 +188,8 @@ fn test_bouncer_try_update(
     use crate::transaction::objects::TransactionResources;
 
     let state = &mut test_state(&BlockContext::create_for_account_testing().chain_info, 0, &[]);
-    let mut transactional_state = TransactionalState::create_transactional(state);
+    let mut transactional_state: TransactionalState<'_, _, VisitedPcsSet> =
```

Comment:
There should likely be a type alias for this type.

Reply:
I have created a dedicated function `TransactionalState::create_transactional_for_testing`.